test/unittest/ndarray.jl

```diff
@@ -7,8 +7,8 @@ using ..Main: rand_dims, reldiff
 ################################################################################
 # Test Implementations
 ################################################################################
-function rand_tensors{N}(dims::NTuple{N, Int})
-  tensor = rand(mx.MX_float, dims)
+function rand_tensors{N}(dims::NTuple{N, Int}, float_type=mx.MX_float)
```

Comment:
A more Julian version of this would be:
```
rand_tensors{N}(dims::NTuple{N, Int}) = rand_tensors(mx.MX_float, dims)
function rand_tensors{N,T}(::Type{T}, dims::NTuple{N, Int})
```